accord-core/src/test/java/accord/impl/basic/Cluster.java

```diff
@@ -274,6 +278,84 @@ public void now(Runnable run)
         run.run();
     }
 
+    public static Map<MessageType, Cluster.Stats> run(Supplier<RandomSource> randomSupplier,
```

Comment:
This was pulled out of `accord-core/src/test/java/accord/local/CommandsTest.java` so another test could use.